.github/workflows/post-commit.yml

```diff
@@ -35,6 +35,26 @@ jobs:
       - uses: ./.github/buildenv
         with:
           args: ./hadoop-ozone/dev-support/checks/build.sh
+  bats:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
In the future I am planning to improve all the existing tests:

 1. With adding name to all the steps
 2. Use upper case for the first letter (default for github actions if the name is missing).

This is the same as the others, so I think we can merge it as is.